src/Components/Server/src/CircuitOptions.cs

```diff
@@ -52,8 +51,18 @@ public class CircuitOptions
         /// exceptions to JavaScript.
         /// </summary>
         /// <remarks>
-        /// The default value for this property is <c>false</c>. This value should only be turned on in development scenarios as turning it
-        /// on in production might result in a sensitive information leak.</remarks>
+        /// This value should only be turned on in development scenarios as turning it on in production might result in the leak of
+        /// sensitive information to untrusted parties.
+        /// </remarks>
+        /// <value>Defaults to <c>false</c>.</value>
         public bool DotNetInteropExceptionDetails { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value that indicates how long the server will wait before timing out an asynchronous JavaScript function invocation.
+        /// </summary>
+        /// <value>
+        /// Defaults to <c>1 minute</c>.
+        /// </value>
+        public TimeSpan DefaultAsyncJavaScriptInteropCallTimeout { get; set; } = TimeSpan.FromMinutes(1);
```

Comment:
I suggest aligning the terminology between these two new options.

For example, if the other one was called `JSInteropDetailedErrors`, then this one could be `DefaultJScriptInteropCallTimeout`. We don't really need `Async` in the option name, because all JSInterop calls from the server are async.

Reply:
Sounds good

Reply:
done